command_helper/workspace_objects.py

```diff
@@ -1,6 +1,6 @@
 from os import listdir, path
 try:
-    from noralize_cell import get_data_from_json
+    from .noralize_cell import get_data_from_json

```

Comment:
`noralize` -> `normalize` ?

Reply:
It's noralize_cell from the first place. I haven't changed it but as you spot the typo. I'll make the change.